src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -60,6 +66,14 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
   );
 };
 
+ItemUser.propTypes = {
+  photo: PropTypes.string,
+  username: PropTypes.string,
+  followed: PropTypes.any,
+  onPress: PropTypes.func,
+  userid: PropTypes.string,
+  isAnon: PropTypes.string
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding PropTypes is a good practice for type checking in React. It helps to catch bugs by validating the types of props passed to components. However, as mentioned earlier, the `isAnon` prop should be a boolean, not a string.


<!-- This is an auto-generated comment by OSS CodeRabbit -->